src/utils/resources/vmi/utils/guest-agent.ts

```diff
@@ -1,6 +1,16 @@
-import { V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import {
+  V1VirtualMachineInstance,
+  V1VirtualMachineInstanceGuestAgentInfo,
+} from '@kubevirt-ui/kubevirt-api/kubevirt';
 
 export const isGuestAgentConnected = (vmi: V1VirtualMachineInstance): boolean =>
   vmi?.status?.conditions?.some(
     (condition) => condition?.type === 'AgentConnected' && condition?.status === 'True',
   );
+
+export const getOsNameFromGuestAgent = (
+  guestAgentData: V1VirtualMachineInstanceGuestAgentInfo,
+): string =>
+  guestAgentData.os.name.includes('Windows') && guestAgentData.os.version.includes('Windows')
```

Comment:
Maybe better `guestAgentData?.os?.name?.includes('Windows')` ? If we don't want to let it break intentionally in case the data would be undefined by some chance..

Reply:
fixed, thanks :+1: :smile: